apps/user-office-backend/src/models/questionTypes/DynamicMultipleChoice.ts

```diff
@@ -50,4 +67,68 @@ export const dynamicMultipleChoiceDefinition: Question = {
         );
     }
   },
+  externalApiCall: async (record: any) => {
+    let recordWithApiCall;
+
+    try {
+      const resp = await axios.get(record.config.url, {
+        headers: {
+          Authorization: `Bearer ${record.config?.token}`,
+        },
+      });
+      const arrayData =
+        Array.isArray(resp.data) &&
+        resp.data.every((el) => typeof el === 'string');
+      if (arrayData) {
+        recordWithApiCall = {
+          ...record,
+          config: {
+            ...record.config,
+            options: resp.data,
+          },
+        };
+      } else {
+        try {
+          //TODO: if jsonPathFilteredData is empty, then we know
```

Comment:
Is there still something todo here or this is just a NOTE comment.

Reply:
I was thinking we might possibly want to add error text to frontend to notify users that jsonpath syntax is possibly wrong. But I think This TODO can be removed.